design-system-docs/src/_component_examples/_text_input/fixed_widths.erb

```diff
@@ -0,0 +1,31 @@
+---
+title: fixed widths
+---
+
+<%= render(CitizensAdviceComponents::TextInput.new(
+  width: :two_chars,
+  name: "example-input-fixed-width-two_chars",
+  label: "Example input", type: :text,
+  options: { hint: "This input is two_chars wide" }
+)) %>
+
+<%= render(CitizensAdviceComponents::TextInput.new(
```

Comment:
Nitpick but this input and the one below (on line 19) are the same width. I think this one should be `four_chars`